measures/ResStockArguments/measure.rb

```diff
@@ -41,7 +41,12 @@ def arguments(model)
       # Following are arguments with the same name but different options
       next if arg.name == 'geometry_unit_cfa'
 
-      args << arg
+      # Convert optional arguments to string arguments that allow Constants.Auto for defaulting
+      if !arg.required
+        args << OpenStudio::Measure::OSArgument.makeStringArgument(arg.name, false)
+      else
+        args << arg
+      end
```

Comment:
This allows options_lookup.tsv to continue to support "auto" for optional arguments.